interface/xtreemfs/MRC.proto

```diff
@@ -753,7 +773,7 @@ service MRCService {
   };
 
   // renews an existing capability to extend the validity period
-  rpc xtreemfs_renew_capability(XCap) returns(XCap) {
+  rpc xtreemfs_renew_capability(xtreemfs_renew_capabilityRequest) returns(XCap) {
```

Comment:
I am not happy with changing the signature of an existing operation, this will definitively break the compatibility with old client releases. We should discuss this issue face-to-face.